cdk/package.json

```diff
@@ -5,27 +5,29 @@
   "scripts": {
     "build": "tsc",
     "test": "jest",
+    "test-update": "jest -u",
     "format": "prettier --write \"{lib,bin}/**/*.ts\"",
     "lint": "eslint lib/** bin/** --ext .ts --no-error-on-unmatched-pattern",
     "synth": "cdk synth --path-metadata false --version-reporting false",
     "diff": "cdk diff --path-metadata false --version-reporting false"
   },
+  "license": "ISC",
```

Comment:
I think we can omit this property.

Reply:
Ah, right, that was on my forgotten list of questions to ask, I deleted it.